lua-mode.el

```diff
@@ -769,6 +769,9 @@ Groups 6-9 can be used in any of argument regexps."
       (setq mode-popup-menu
             (cons (concat mode-name " Mode Commands") lua-emacs-menu)))
 
+  (make-variable-buffer-local 'completion-at-point-functions)
```

Comment:
For making other people's variables local just in this mode it's better to use `make-local-variable`.